src/Okta.Sdk/AddEmailFactorOptions.cs

```diff
@@ -14,5 +14,10 @@ public sealed class AddEmailFactorOptions
         /// Gets or sets the email
         /// </summary>
         public string Email { get; set; }
+
+        /// <summary>
+        /// Gets or sets the lifetime of the OTP in a range 1 to 86400, 300 seconds is the default.
```

Comment:
Instead of describing the range in the comments, I suggest we add a [`Remarks`](https://docs.microsoft.com/en-us/dotnet/csharp/codedoc#remarks) (or `<see>`/`<seealso>`) section with a link to the docs (for example, `<Remarks>For more details see [link]</Remarks>`. Since our team doesn't own this API, if the API team decides to change the range we will need to release a new version just to update the comments. This is a general rule for methods/attributes documentation. 
Defaults are OK since they generally explicitly set in our code.